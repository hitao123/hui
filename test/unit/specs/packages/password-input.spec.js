import { shallowMount } from '@vue/test-utils';
import passwordInput from '../../../../packages/password-input';

describe('Dialog component', () => {
  it('should show Dialog', () => {
    const warpper = shallowMount(passwordInput, {
      propsData: {
        value: '1234'
      }
    });
    expect(warpper.html()).toMatchSnapshot();
  });
});
