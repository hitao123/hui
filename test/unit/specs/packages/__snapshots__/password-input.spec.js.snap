// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog component should show Dialog 1`] = `
<div class="h-password-input">
  <ul class="h-hairline--surround h-password-input__security">
    <li class="h-hairline"><i style="visibility: visible;"></i></li>
    <li class="h-hairline"><i style="visibility: visible;"></i></li>
    <li class="h-hairline"><i style="visibility: visible;"></i></li>
    <li class="h-hairline"><i style="visibility: visible;"></i></li>
    <li class="h-hairline"><i style="visibility: hidden;"></i></li>
    <li class="h-hairline"><i style="visibility: hidden;"></i></li>
  </ul>
</div>
`;
